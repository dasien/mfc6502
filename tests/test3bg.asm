; Perform simple addition and test transfers to from registers and stack.

LDA #01
PHA
LDA #02
PHA
PLA
TAX
PLA
TAY
TXA
STY $01
ADC $01