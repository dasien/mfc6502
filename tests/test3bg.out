A9 01
48 
A9 02
48 
68 
AA 
68 
A8 
8A 
84 01
65 01
